python/cuxfilter/charts/deckgl/deckgl.py

```diff
@@ -70,7 +70,7 @@ def choropleth(
 
     geo_color_palette: bokeh.palette,  default bokeh.palettes.Inferno256
 
-    nan_color: str, default white
+    nan_color: hex color code, default '#d3d3d3'
```

Comment:
Seen this hex code a few places now, maybe make it a module level `CUXF_NAN_COLOR` somewhere convenient?

Reply:
added CUXF_NAN_COLOR as a module level constant in cuxfilter.charts.constants